### ISSUE TITLE
Potential live-connection issue with Firefox

### ISSUE THREAD

**Issue by erichiggins** [firebase/emberfire#543]
Following-up with a ticket based on my last comment on #534
When browser-testing my app, I noticed that the live-connection does not work as expected in Firefox (58.0.1 64-bit on OSX). I thought it might be related.

### Version info

```
e.g.
DEBUG: -------------------------------
DEBUG: Ember      : 2.16.2
DEBUG: Ember Data : 2.16.3
DEBUG: Firebase   : 3.9.0
DEBUG: EmberFire  : 2.0.8
DEBUG: jQuery     : 3.2.1
DEBUG: -------------------------------
```
**Note:** The EmberFire version data above came back as `0.0.0`, so I filled it in manually.

### Test case

Not sure I can do this for you since it requires a new Firebase app and I've already maxed out the number I'm allowed to create.

To summarize though, you'd just need a list-view in an Ember app that displays Firebase Realtime database results.

### Steps to reproduce

1. Open the list-view of your Ember app in Firefox.
2. In the Firebase console inside another tab, add a new result to the Realtime Database that should affect the list-view.

### Expected behavior

Expect to see the list-view automatically update and display the new database entry.

### Actual behavior

The list-view does not update automatically.

If you navigate forward/backward, or reload, it will force it to refresh and display as expected.
